identity/app/views/profile/privacyForm.scala.html

```diff
@@ -76,19 +76,53 @@ <h2 class="form__heading">What can we tell you about?</h2>
     </form>
 }
 
+
 @channelConsentForm = {
     <form method="post" action="@idUrlBuilder.buildUrl("/privacy/edit", idRequest)" novalidate>
         @views.html.helper.CSRF.formField
-        <div class="manage-account__switches">
+        <div class="manage-account__switches manage-account__switches--single-column">
+
+            <label class="label">Please tick the boxes below to let us know how you’d prefer to hear from us.</label>
             <ul>
             @helper.repeatWithIndex(privacyForm("consents"), min=1) { (consentField, index) =>
                 @if(isSmsChannel(consentField, user)) {
                   <li>
-                      @fragments.consentSwitch(consentField)(messages)
+                      @fragments.form.switch(
+                          title = "",
+                          subheading = None,
+                          description = Some("Yes please, I would like to receive updates about the products and services I’ve selected above <b>by SMS</b> (text messages)"),
```

Comment:
we really shouldn't be duplicating this copy for compliance reasons, we should pull it from `identity`. AFAIk this is only being done because of the bold text?